sections/git_committime.zsh

```diff
@@ -0,0 +1,62 @@
+#
+# Git commit_time
+#
+# Show time since previous commit
+# Inspired from "hyperzsh" theme
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+
+SPACESHIP_GIT_COMMITTIME_SHOW="${SPACESHIP_GIT_COMMITTIME_SHOW:=true}"
+SPACESHIP_GIT_COMMITTIME_PREFIX="${SPACESHIP_GIT_COMMITTIME_PREFIX:=since }"
+SPACESHIP_GIT_COMMITTIME_SUFFIX="${SPACESHIP_GIT_COMMITTIME_SUFFIX:=}"
+SPACESHIP_GIT_COMMITTIME_THRESHOLD_LOW="${SPACESHIP_GIT_COMMITTIME_THRESHOLD_LOW:=30}" # in minutes
+SPACESHIP_GIT_COMMITTIME_THRESHOLD_HIGH="${SPACESHIP_GIT_COMMITTIME_THRESHOLD_HIGH:=240}" # in minutes
+
+# ------------------------------------------------------------------------------
+# Section
+# ------------------------------------------------------------------------------
+
+spaceship_git_committime() {
+    [[ $SPACESHIP_GIT_COMMITTIME_SHOW == false ]] && return
```

Comment:
for readability maybe use underscore, `spaceship_git_commit_time` and `$SPACESHIP_GIT_COMMIT_TIME_SHOW` ?